test/controllers/songs_controller_test.rb

```diff
@@ -1,19 +1,22 @@
-require 'test_helper'
+require "test_helper"
```

Comment:
Just curious: is there a ruby standard or convention that says to use " over ' in these cases?